.github/workflows/lintcheck.yml

```diff
@@ -115,4 +115,17 @@ jobs:
       uses: actions/download-artifact@v4
 
     - name: Diff results
-      run: ./target/debug/lintcheck diff {base,head}/ci_crates_logs.json >> $GITHUB_STEP_SUMMARY
+      # GH's summery has a maximum size of 1024k:
+      # https://docs.github.com/actions/using-workflows/workflow-commands-for-github-actions#adding-a-markdown-summary
+      # That's why we first log to file and then to the summary and logs
+      run: |
+        ./target/debug/lintcheck diff {base,head}/ci_crates_logs.json >> full_diff.md
+        ./target/debug/lintcheck diff {base,head}/ci_crates_logs.json --truncate >> truncated_diff.md
+        head -c 1024000 truncated_diff.md >> $GITHUB_STEP_SUMMARY
+        cat truncated_diff.md
```

Comment:
Is `cat`ing the `diff` still helpful, when printing it to the summary AND uploading it?

Reply:
It would be my first place to look, when I notice that the summary has been truncated. It's just faster than downloading the file. The artifact currently doesn't include the truncated diff. I'll add it, 1 sec

Reply:
Ah, in that case did you mean to `cat` the `full_diff.md` instead of the `truncated_diff`?

Reply:
Now it makes sense :+1: